src/format-files.js

```diff
@@ -49,9 +51,10 @@ function formatFilesFromArgv({
     prettierLast,
     prettierOptions,
   }
-
+  const configContext = new Config({}, new Linter())
```

Comment:
Also, it looks like `configContext` is only used within the `if` statement below, could we move this declaration to within that `if` statement?